clash-prelude/src/Clash/XException.hs

```diff
@@ -397,6 +398,8 @@ printX :: ShowX a => a -> IO ()
 printX x = putStrLn $ showX x
 
 instance ShowX ()
+-- @since 1.8.2
```

Comment:
To be picked up by Haddock it needs to be:

```suggestion
-- | @since 1.8.2
```

Right?

Reply:
Good catch!